src/site/layouts/office.drupal.liquid

```diff
@@ -251,9 +251,6 @@
                 tabindex="0">Prev</button>
             </span>
             <div id="pager-nums-insert" class="va-pagination-inner">
-              <button id="va-pagination-active-num"
```

Comment:
Where was this button's functionality hooked up? Has that also been removed? (Sorry if that's a silly question; I don't have a lot of context.)

Reply:
In libraryPagerGen() @cvalarida